client/src/pages/FlightData/index.js

```diff
@@ -107,6 +113,7 @@ const FlightData = () => {
 		time: ["Time Loiter", "Time Loiter"],
 		jump: ["Jump", "Jump"],
 		uav: ["UAV", "UAV Location"],
+		uavPath: ["UAV Path", "UAV Path"],
```

Comment:
Again, you should use custom datatypes to populate this

Reply:
You did in fact implement custom datatypes above (great job!), you just never added display attributes for them. This is what I'm looking for in `FlightMap.js`.